app/core/Tracker.coffee

```diff
@@ -136,6 +136,14 @@ module.exports = class Tracker extends CocoClass
         eventAction: action
       gaFieldObject.eventLabel = properties.label if properties.label?
       gaFieldObject.eventValue = properties.value if properties.value?
+
+      # Adding label for tracking home page A/B testing group on GA for analyzing results
+      # TODO: what if gaFieldObject.label already exists and we still want to track a/b results
+      # As of now gaFieldObject.label doesnt exist for the specific events which are relevant for a/b test results, so not a concern for now
+      # Probably add multiple labels as keys in gaFieldObject.label in the future, if required.
+      if properties.trackABResult and not gaFieldObject.label and me.getHomePageTestGroup()
+        gaFieldObject.eventLabel = "{testGroup:"+ me.getHomePageTestGroup() + "}"  # {testGroup:A} / {testGroup:B} / {testGroup:C}
```

Comment:
@AndrewJakubowicz Does the TODO comment make more sense now? I also changed the format of `eventLabel`.

Reply:
Yeah this is great!